apps/frontend/src/app/3_organisms/ZeroLocForm/AdjustCreditLine.tsx

```diff
@@ -101,8 +103,10 @@ export const AdjustCreditLine: FC<AdjustCreditLineProps> = ({
 
   const { value: creditWeiBalance } = useAssetBalance(debtToken);
 
-  const { value: maxCollateralWeiAmount } = useAssetBalance(
+  const { value: maxCollateralWeiAmount } = useMaxAssetBalance(
     SupportedTokens.rbtc,
+    getRskChainId(),
+    collateralType === AmountType.Add ? GAS_LIMIT_OPEN_TROVE : 0,
```

Comment:
```suggestion
    collateralType === AmountType.Add ? GAS_LIMIT_OPEN_TROVE : undefined,
```

Reply:
`0` is correct for this particular case, because gas fee should not be removed for withdrawing collateral (because rbtc balance is taken not from user wallet, but withdrawing from contract)